doc/howto/config.rst

```diff
@@ -408,6 +408,20 @@ Example::
     }
 
 
+want_response_signed
+""""""""""""""""""""
+
+Indicates that Authentication Responses to this SP should be signed.
```

Comment:
I would say "must be signed" . And/or we can clarify that if this is set to True, the SP will not consume any SAML Responses that are not signed.